addons/mail/static/src/core/common/persona_model.js

```diff
@@ -97,6 +96,9 @@ export class Persona extends Record {
             }
         },
     });
+    /** @type {string} */
+    im_status_access_token;
```

Comment:
We actually run into the problem that the compute won't be retriggered if the token is known after the user is already added in `storeAsTrackedImStatus`.

We might need an `onUpdate` on `im_status_access_token` to also update the bus service channel with the new token.

Reply:
Or make `presenceChannel` itself a compute and have the onUpdate on it.